lib/driver-cli.js

```diff
@@ -0,0 +1,249 @@
+import _ from 'lodash';
+import path from 'path';
+import { KNOWN_DRIVERS } from './drivers';
+import DriverConfig, { INSTALL_TYPE_NPM, INSTALL_TYPE_GIT, INSTALL_TYPE_GITHUB,
+                       INSTALL_TYPE_LOCAL } from './driver-config';
+import NPM from './npm';
+import { errAndQuit, log, spinWith, JSON_SPACES } from './cli-utils';
+import { util, fs } from 'appium-support';
+
+async function runDriverCommand (args) {
```

Comment:
it would be nice to have some docstring on that. It is not very obvious what `args`is

Reply:
yep, will explain